build.gradle

```diff
@@ -56,7 +56,10 @@ dependencies {
     compile("com.github.springfox.loader:springfox-loader:${springfoxLoaderVersion}")
 
     compile('org.springframework.boot:spring-boot-starter-web')
-    runtime('org.springframework.boot:spring-boot-actuator')
+    compile('org.springframework.boot:spring-boot-starter-actuator')
+    
+    compile('io.micrometer:micrometer-registry-prometheus:latest.release')
```

Comment:
What version is `latest.release` ?

Reply:
It's simply the best. Better than all the rest.

Reply:
Or registry-prometheus = 1.6.4 and spring-legacy = 1.3.17.